lib/Nodes/Node.php

```diff
@@ -171,6 +176,17 @@ protected function getRenderer(): NodeRenderer
         return $this->createDefaultRenderer();
     }
 
+    private function countStartingWhitespace(string $line): int
+    {
+        for ($k = 0; $k < strlen($line); $k++) {
+            if (trim($line[$k]) !== '') {
+                break;
+            }
+        }
+
+        return $k;
+    }
```

Comment:
Can you think of a reason why this for loop was used, instead of using `strlen($line) - strlen(ltrim($line))` to calculate the indent size?